node/network/tests/integration_test.rs

```diff
@@ -150,6 +150,7 @@ fn test_block_catch_up_from_start() {
 // this test is expensive to run. should be ignored when running
 // all tests
 #[test]
+#[ignore]
```

Comment:
Should every [ignore] test have an issue associated with it too? @azban

Reply:
no.. ignore is a legitimately used in rust (https://doc.rust-lang.org/1.5.0/book/testing.html#the-ignore-attribute). we should probably add `--ignored` in ci though if these are legitimate.